isis/tests/TestCsmModel.cpp

```diff
@@ -568,6 +569,8 @@ csm::EcefCoord TestCsmModel::imageToGround(const csm::ImageCoord& imagePt,
 
   double lon = center_longitude + (imagePt.samp - getImageSize().samp/2.0)/scale;
   double lat = center_lat + (imagePt.line - getImageSize().line/2.0)/scale;
+  lon*=M_PI/180;
+  lat*=M_PI/180;
```

Comment:
Add spaces around `*=`

Reply:
Thanks for catching and fixing the degrees <-> radians issue!